### ISSUE TITLE
Contribute skip attribute to linkinator

### ISSUE THREAD

**Issue by davifantasia** [commercetools/commercetools-docs-kit#659]

**Comment by davifantasia** [comment]
Will wait for outcome of discussion in linkinator repository here - https://github.com/JustinBeckwith/linkinator/issues/175

**Comment by nkuehn** [comment]
https://github.com/JustinBeckwith/linkinator/pull/180  PR with as a try to convince them

**Comment by AgaMillward** [comment]
@nkuehn is this ticket still valid for us? it has been on hold since September?

**Comment by nkuehn** [comment]
Yes it is.  If someone from the team can take the time to adjust the PR I made to linkinator to the tool's current codebase (i.e. "rebase" and fix it) we can ask the maintainer again (or even decide to use our own fork of it).

**Comment by AgaMillward** [comment]
How long shall we keep a ticket waiting? Agree with the team. Shall we remove tickets if they sit for a few months with no action?
